src/main/java/org/broadinstitute/hellbender/tools/copynumber/CollectReadCounts.java

```diff
@@ -124,16 +123,21 @@
      */
     private SampleLocatableMetadata metadata;
 
+    private List<SimpleInterval> intervals;
+
+    private String currentContig = null;
+    private boolean isCoordinateSorted;
+
     /**
      * Overlap detector used to determine when read starts overlap with input intervals.
      */
     private CachedOverlapDetector<SimpleInterval> intervalCachedOverlapDetector;
 
-    private Multiset<SimpleInterval> intervalMultiset = HashMultiset.create();
+    private Multiset<SimpleInterval> intervalMultiset;
 
     @Override
     public List<ReadFilter> getDefaultReadFilters() {
-        final List<ReadFilter> filters = new ArrayList<>(super.getDefaultReadFilters());
+        final List<ReadFilter> filters = new ArrayList<>();
```

Comment:
See #5233.  I don't know if we should disable this by default, or just do it at the WDL level for the production WDL.  Not sure if we should also disable over in CollectAllelicCounts or if that will have any effects.  With the cost reductions from the other fixes, I don't think it's absolutely necessary to disable the filter (although we should rerun cost estimates to be sure memory requirements don't change).  Up to you.

Reply:
I think we should keep it enabled by default, since that's the expected behavior for all GATK tools, but disable it from the WDL (and make it part of best practices).